### ISSUE TITLE
0.7.0 fails even if there is no error

### ISSUE THREAD

**Issue by ASarco** [saadmk11/github-actions-version-updater#31]
When running this action, the workflows fails, even if there's no evident error and all the actions are checked:

![image](https://user-images.githubusercontent.com/9863532/197978458-b97b6f15-6707-46d0-8e06-51584ac9cb47.png)

The job is configured like this:
```yaml
jobs:
  build:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
        with:
          # Access token with `workflow` scope is required
          token: ${{ secrets.ACTION_GITHUB_TOKEN }}

      - name: Run GitHub Actions Version Updater
        uses: saadmk11/github-actions-version-updater@v0.7.0
        with:
          # Optional, This will be used to configure git
          # defaults to `github-actions[bot]` if not provided
          #committer_username: 'test'
          #committer_email: 'test@test.com'
          # Optional, allows customizing the commit message and pull request title
          # Both default to 'Update GitHub Action Versions'
          commit_message: 'Github Actions versions have changed'
          pull_request_title: 'Github Actions versions have changed'
          # Access token with `workflow` scope is required
          token: ${{ secrets.ACTION_GITHUB_TOKEN }}
          release_types: major
```

**Comment by saadmk11** [comment]
Hi @ASarco, Thanks for reporting the issue. I have tried to reproduce this using the same configuration but I did not see this error. 

Can you please provide some more details?

Such as: 

- Is the repository private or public? 
- Did this happen only once or every time?

If possible can you provide the full raw log?

**Comment by ASarco** [comment]
Hi @saadmk11,
Yes it happens every time. 
It's a private repo.
It used to work fine with the previous version 0.5.6, but I needed to be alerted only on major releases. 

Happy to provide you with the full raw log, bout how do I get it?

**Comment by saadmk11** [comment]
> Hi @saadmk11, Yes it happens every time. It's a private repo. It used to work fine with the previous version 0.5.6, but I needed to be alerted only on major releases.
> 
> Happy to provide you with the full raw log, bout how do I get it?

Click on **View Raw Logs** on the Jobs page.

![Screenshot](https://user-images.githubusercontent.com/24854406/197993843-2fc8db43-43c0-475f-a8cd-752fdade3e32.png)

**Comment by ASarco** [comment]
[4_Run GitHub Actions Version Updater.txt](https://github.com/saadmk11/github-actions-version-updater/files/9868620/4_Run.GitHub.Actions.Version.Updater.txt)
Looking at that log, seems like the error is:
`FileNotFoundError: [Errno 2] No such file or directory: '.github/workflows/kubernetes-test.yaml'`
But there is no such a file in .github/workflows/ :
![image](https://user-images.githubusercontent.com/9863532/198005583-122c3cba-b04b-4725-9f19-349889c402e8.png)
Where is the action picking the .yaml files from?

**Comment by saadmk11** [comment]
> [4_Run GitHub Actions Version Updater.txt](https://github.com/saadmk11/github-actions-version-updater/files/9868620/4_Run.GitHub.Actions.Version.Updater.txt) Looking at that log, seems like the error is: `FileNotFoundError: [Errno 2] No such file or directory: '.github/workflows/kubernetes-test.yaml'` But there is no such a file in .github/workflows/ : ![image](https://user-images.githubusercontent.com/9863532/198005583-122c3cba-b04b-4725-9f19-349889c402e8.png) Where is the action picking the .yaml files from?

The Action is Using GitHub's List repository workflows API https://docs.github.com/en/rest/actions/workflows#list-repository-workflows

We probably need to skip the file if it does not exist.

**Comment by ashishb** [comment]
Same issue in the private repo with a file that no longer exists.

**Comment by saadmk11** [comment]
This bug should be fixed on [v0.7.1](https://github.com/saadmk11/github-actions-version-updater/releases/tag/v0.7.1). Please try it out and re-open the issue if the bug exists. :)
